common-items.lic

```diff
@@ -548,6 +548,12 @@ module DRCI
     DRC.rummage('B', container)
   end
 
+  # Takes in the noun of the configured necro material stacker, and returns the current material item count.
+  def count_necro_stacker(necro_stacker)
+    count = DRC.bput("study my #{necro_stacker}", /currently holds \d+ items/).scan(/\d+/).first.to_i
```

Comment:
You could return the `bput` directly, without declaring count

Reply:
Thank you! :) Done it that way.